x-pack/plugins/security/server/routes/telemetry/authentication_type.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createHash } from 'crypto';
+
+import { schema } from '@kbn/config-schema';
+
+import type { RouteDefinitionParams } from '..';
+import { wrapIntoCustomErrorResponse } from '../../errors';
+import { createLicensedRouteHandler } from '../licensed_route_handler';
+
+const COUNTER_TYPE = 'SecurityAuthType';
+
+export function defineTelemetryOnAuthTypeRoutes({
+  getAuthenticationService,
+  router,
+  usageCounter,
+}: RouteDefinitionParams) {
+  router.post(
+    {
+      path: '/internal/security/telemetry/auth_type',
```

Comment:
Nit: can we name this something a little more descriptive?
Taking into account my earlier comment about "usage collection" vs "telemetry", what do you think of this?

```suggestion
      path: '/internal/security/usage/record_auth_type',
```